components/HooksEditor.tsx

```diff
@@ -200,15 +200,15 @@ const HooksEditor = () => {
             defaultValue={file?.content}
             // onChange={val => (state.files[snap.active].content = val)} // Auto save?
             beforeMount={monaco => {
-              if (!snap.editorCtx) {
-                snap.files.forEach(file =>
-                  monaco.editor.createModel(
-                    file.content,
-                    file.language,
-                    monaco.Uri.parse(`file:///work/c/${file.name}`)
-                  )
-                )
-              }
+              // if (!snap.editorCtx) {
+              //   snap.files.forEach(file =>
+              //     monaco.editor.createModel(
+              //       file.content,
+              //       file.language,
+              //       monaco.Uri.parse(`file:///work/c/${file.name}`)
+              //     )
+              //   )
+              // }
 
```

Comment:
@valstu is this necessary for anything? It created models for initial list of files, but everything works without it also, so had it commented! Did I broke anything? 😅

Reply:
Hmm.. good question...  IIRC this is related to thing that in some cases the all the files need to initializedso we can reference them in another file. But with C files this is not a problem since language server is handling this, it might be problem with TypeScript files though, but I am not 100% sure.  But if compiling works fine and we don't have any squiggly lines complaining about unknown imports or refrences I think we should be ok.